tulflow/harvest.py

```diff
@@ -153,7 +153,7 @@ def append(self, record):
         self.root.append(record)
 
     def tostring(self):
-       return etree.tostring(self.root).decode("utf-8")
+        return etree.tostring(self.root, encoding="utf-8").decode("utf-8")
```

Comment:
```suggestion
       return etree.tostring(self.root, encoding="utf-8").decode("utf-8")
```

Reply:
Adding the space brings it in line with the other code in the page.  That's why I added it.